setup.py

```diff
@@ -154,6 +154,7 @@ def get_long_description():
               "Operating System :: POSIX",
               "Topic :: Software Development :: Quality Assurance",
               "Topic :: Software Development :: Testing",
+              "Programming Language :: Python :: 2.7",
```

Comment:
yes, it's important to be clear and specific.

Reply:
Right, since py3 support is not fully done yet, this looks good.